flax/__init__.py

```diff
@@ -32,4 +32,4 @@
 
 # Allow `import flax`; `flax.nn.[...]`, and the same for `flax.optim.[...]`
 from . import nn
-from . import optim
+from . import optim
```

Comment:
Please don't change unnecessary files.

Reply:
(other than this, I think this PR is ready to merge.)

Reply:
Fixed! (Sorry, this came from my confusion when trying to run the imagenet example)